include/avif/avif.h

```diff
@@ -936,6 +1040,7 @@ typedef struct avifDecoder
     // If you don't actually leverage this data, it is best to ignore it here.
     avifBool ignoreExif;
     avifBool ignoreXMP;
+    // TODO(maryla): add flags to ignore the main image, or ignore the gain map if desired.
```

Comment:
Optional: Move this TODO comment to the end of the struct to reduce the chance of accidentally adding the new flags here.